src/Radarr.Api.V3/swagger.json

```diff
@@ -2,15 +2,12 @@
   "openapi": "3.0.0",
   "info": {
     "title": "Radarr",
-    "description": "Movie Automation",
-    "contact": {
-      "url": "https://radarr.video"
-    },
-    "version": "3.0.0"
+    "version": "3.0.0",
+    "description": "The way users should interact with Radarr programatically. To utilize any of these endpoints you will need a few pieces of information:\n\n* hostPath: localhost, 10.1.0.1, 192.168.1.1, etc\n* hostPort: 7878 (unless you modify it)\n* apiKey: Located in Settings > General > Security"
   },
   "servers": [
     {
-      "url": "{protocol}://{hostPath}/api/v3",
+      "url": "{protocol}://{hostPath}:{hostPort}/api/v3",
```

Comment:
`hostPort` is not needed, its part of `hostPath` in the default below... most will change it to a revproxied url to test which will not have a port. If testing with a port just include it in the `hostPath` like the example has it